dictionaries/gis/README.md

```diff
@@ -0,0 +1,50 @@
+# CSpell GIS and PostGIS Dictionary
+
+GIS and PostGIS dictionary for cspell.
```

Comment:
Hi

I have no idea what is GIS or PoatGIS

so adding a short explanation and links could be interesting

Thanks

Reply:
I added what GIS stands for and a link to PostGIS.